mkdocs.yml

```diff
@@ -0,0 +1,2 @@
+site_name: Daml on Besu
```

Comment:
not Daml on Besu

Reply:
Good catch thanks, corrected and CODEOWNERS file added